scripts/tests/test_validate_docstrings.py

```diff
@@ -720,6 +726,13 @@ def test_bad_class(self):
         assert isinstance(errors, list)
         assert errors
 
+    @capture_stderr
+    def test_bad_class(self):
```

Comment:
Hmm most of our focus in on docstrings for methods. Not sure we explicitly require the class to have a doctoring though. @datapythonista

Reply:
I forgot to rename function i've been using as an example. will fix